middlewares/authzCourse.sql

```diff
@@ -1,3 +1,20 @@
+-- BLOCK insert_xc101_viewer
+WITH
+example_course AS (
+    SELECT * FROM pl_courses WHERE (options->'isExampleCourse')::boolean IS TRUE
+)
+INSERT INTO course_permissions (user_id, course_id, course_role)
+    SELECT
+        cp.user_id, xc.id, 'Viewer'
+    FROM
+        course_permissions AS cp,
+        example_course AS xc
```

Comment:
Will this do an unnecessary cross join? Should I just remove this `example_course as xc` line and change the `WHERE` clause to `example_course.id = $course_id`?